script/main.go

```diff
@@ -38,6 +38,9 @@ func main() {
 	case "review":
 		reviewer := Reviewer{}
 		reviewer.Review()
+	case "approve":
```

Comment:
[Future suggestion] You might consider using spf13/cobra and spf13/pflag, as a way to structure your “main” function, depending on how many more commands and flags. Having a dependency to maintain could be more overhead than this tool deserves, but it could give it the structure it needs.